test/fast/Running 'nvm install --save --lts' with incorrect file permissions fails nicely

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -e
```

Comment:
Most tests have `set -ex` which makes sense generally, but `nvm install` makes huge amounts of output with this enabled, do you think `set -e` is sufficient?

Reply:
i think that's fine; the `x` is to make debugging easier when it _does_ fail.